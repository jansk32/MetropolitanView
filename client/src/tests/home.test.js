
import React from 'react';
import HomePage from '../homePage'
import renderer from 'react-test-renderer';


test("Testing a Home Page", () => {
    const component = renderer.create(
        <HomePage />
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

})