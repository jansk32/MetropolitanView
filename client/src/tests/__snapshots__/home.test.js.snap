// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing a Home Page 1`] = `null`;
